tests/trezor/daemon.cpp

```diff
@@ -313,7 +313,7 @@ void mock_daemon::mine_blocks(size_t num_blocks, const std::string &miner_addres
 {
   bool blocks_mined = false;
   const uint64_t start_height = get_height();
-  const auto mining_timeout = std::chrono::seconds(30);
+  const auto mining_timeout = std::chrono::seconds(120);
```

Comment:
What is the trezor daemon ? I'm surprised to see mining stuff in there.

Reply:
This one was a bit tricky. This is libwallet test, testing the whole stack of the Wallet API which is used by the monero-gui for example.

The test constructs the transaction with Trezor. The I need to check whether the transaction is cosntructed properly. Other tests do that by manipulating blockchain and basically also "mining" a new block so the transaction constructed by Trezor is added to a new block. Then the new blockchain is checked and Trezor transaction is validated as it would be by a normal Monero node on mainnet. 

In Wallet API I don't have direct access to blockchain so I just call mining for a while until the Trezor transaction gets into a new block for validation and transaction checks. 
The test also verifies that the new Trezor transaction is correctly detected by Wallet API on the blockchain.